src/iop/channelmixerrgb.c

```diff
@@ -1723,7 +1723,7 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece,
     if(find_temperature_from_raw_coeffs(&(self->dev->image_storage), custom_wb, &(x), &(y)))
     {
       // Convert illuminant from xyY to XYZ
-      float XYZ[3];
+      float XYZ[4];
```

Comment:
I don't understand this change as we have:

`static inline void illuminant_xy_to_XYZ(const float x, const float y, float XYZ[3])`

Reply:
Because the same XYZ vector is used 2 lines after as a parameter to 
`static inline void convert_any_XYZ_to_LMS(const float XYZ[4], float LMS[4], dt_adaptation_t kind)`
Which could read a 4 float vector.

Reply:
I see !